frame/contracts/src/tests.rs

```diff
@@ -458,7 +458,12 @@ fn compile_module<T>(fixture_name: &str) -> wat::Result<(Vec<u8>, <T::Hashing as
 where
 	T: frame_system::Config,
 {
-	let fixture_path = ["frame/contracts/fixtures/", fixture_name, ".wat"].concat();
+	let fixture_root = if std::env::var("CARGO_MANIFEST_DIR").is_ok() {
```

Comment:
Shouldn't we use `CARGO_MANIFEST_DIR` in case it is set (making the path absolute)?

Reply:
that would be the default behavior happening behind the scene to resolve the absolute path, we could to it explicitly if that's what you mean

Reply:
ah I guess you are write we can getr the value and use it directly here, that's probably cleaner

Reply:
Yes that is what I mean. Use the value instead of calling `is_ok`.